projects/requisition-management/src/app/components/requisition-summary/requisition-summary.component.html

```diff
@@ -0,0 +1,73 @@
+      <dd class="col-6 col-sm-8 col-md-10">
+        <a *ngIf="view === 'buyer'; else noOrderLink" [routerLink]="'/account/orders/' + requisition.id">{{
+          requisition.orderNo
+        }}</a>
+        <ng-template #noOrderLink>{{ requisition.orderNo }}</ng-template>
+      </dd>
+    </ng-container>
+
+    <dt class="col-6 col-sm-4 col-md-2">{{ 'approval.detailspage.order_date.label' | translate }}</dt>
+    <dd class="col-6 col-sm-8 col-md-10">{{ requisition.creationDate | ishDate }}</dd>
+
+    <ng-container *ngIf="view === 'buyer'">
+      <dt class="col-6 col-sm-4 col-md-2">{{ 'approval.detailspage.approver.label' | translate }}</dt>
+      <dd class="col-6 col-sm-8 col-md-10">
+        <ng-container *ngIf="requisition.approval?.statusCode !== 'pending'; else pendingCustomerApprovers">
+          {{ requisition.approval?.approver?.firstName }} {{ requisition.approval?.approver?.lastName }}
+        </ng-container>
+        <ng-template #pendingCustomerApprovers>
+          <ng-container *ngFor="let approver of requisition.approval?.customerApprovers; let i = index">
+            <ng-container *ngIf="i > 0">, </ng-container>{{ approver.firstName }} {{ approver.lastName }}
+          </ng-container>
+          ({{ requisition.approval.status }})
+        </ng-template>
+      </dd>
+    </ng-container>
+
+    <ng-container *ngIf="view === 'approver'">
+      <dt class="col-6 col-sm-4 col-md-2">{{ 'approval.detailspage.purchaser.label' | translate }}</dt>
+      <dd class="col-6 col-sm-8 col-md-10">{{ requisition.user?.firstName }} {{ requisition.user?.lastName }}</dd>
+    </ng-container>
+
+    <ng-container *ngIf="requisition.approval?.statusCode !== 'pending'">
+      <dt
+        *ngIf="requisition.approval?.statusCode === 'approved'; else labelDateRejected"
+        class="col-6 col-sm-4 col-md-2"
+      >
+        {{ 'approval.detailspage.approval_date.label' | translate }}
+      </dt>
+      <ng-template #labelDateRejected>
+        <dt class="col-6 col-sm-4 col-md-2">{{ 'approval.detailspage.rejection_date.label' | translate }}</dt>
+      </ng-template>
+      <dd class="col-6 col-sm-8 col-md-10">{{ requisition.approval.approvalDate | ishDate }}</dd>
+    </ng-container>
+
+    <dt class="col-6 col-sm-4 col-md-2">{{ 'approval.detailspage.order_total.label' | translate }}</dt>
+    <dd class="col-6 col-sm-8 col-md-10">{{ requisition.totals?.total | ishPrice: 'gross' }}</dd>
+
+    <dt class="col-6 col-sm-4 col-md-2">{{ 'approval.detailspage.approval_status.label' | translate }}</dt>
+    <dd class="col-6 col-sm-8 col-md-10">
+      <span
+        class="border border-secondary badge badge-secondary text-capitalize"
+        [ngClass]="{
+          'border-primary badge-primary': requisition.approval.statusCode === 'approved',
+          'border-danger badge-danger': requisition.approval.statusCode === 'rejected'
+        }"
+      >
+        {{ requisition.approval.status }}</span
+      >
+    </dd>
+
```

Comment:
Approval Badges

Please use other badges for the approval status: 
<span class="badge badge-success">Success</span> for APPROVED
<span class="badge badge-danger">Danger</span> for REJECTED
<span class="badge badge-warning">Warning</span> for PENDING